### PR TITLE
AC-649: Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ribhavsharma**
## Description of what I changed
Drafted the Contribution Guidelines for the Android Client to help newcomers get started with contributing to the project.

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-649

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] All new and existing **tests passed**.

- [ ] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* You can make a pull request by heading over to your repository link, where you'll see your latest commit, and a button with the text ```Create Pull Request``` on it.
+* Title your Pull Request as ```[AC-<IssueNumber>] <Short Description of what you did>```
+* Fill all the details in the Pull Request template and finally post the Pull Request.
+
+## Squashing your commits
+It is very common that a PR doesn't get approved on the first try and you are told to do some edits to your code. In a situation like this, your PR is obviously going to have a lot of commits. A large number of commits makes it difficult to manage the commit tree. That's when squashing is going to help you combine all the commits into one single one. You can squash your commits by typing these commands in the Bash:
+
+```git rebase -i HEAD~x``` Please note that x is the number of commits you want to merge.
+
+For example: ```git rebase -i HEAD~2``` opens rebase for the last three commits.
```
Comment (Devanshk9) [inline CONTRIBUTING.md]: hi HEAD~2 WILL OPEN LAST 2 COMMITS NOT 3 COMMITS PLEASE CORRECT IT.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+If the issue is tagged ```Ready for Work```, you may claim it and start working on it.
+
+## Working on your local repository
+* First, open Git Bash and ```cd``` into your repository folder.
+* It is common practice to create a branch for every pull request. 
+* To create one, type ```git checkout -b [branch-name]```  
+* Make the changes to the files that will solve the issue according to you. 
+
+## Performing a Gradle Check (for Android Projects)
+It is always advised to run a gradle check before you create a pull request. You can do this by typing ```./gradlew check``` in your terminal of Android Studio. If the check is failing, please fix all the errors.
```
Comment (Devanshk9) [inline CONTRIBUTING.md]: hi @ribhavsharma 
Kindly also tell in brief about the android studio and give the download link

**Comment by ribhavsharma** [comment]
If someone wishes to contribute to the Android client, I think it's safe to assume that he/she must be knowing what android studio is.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Devanshk9) [reply CONTRIBUTING.md]: What about this. @ribhavsharma

Comment (ribhavsharma) [reply CONTRIBUTING.md]: If someone wishes to contribute to the Android client, I think it's safe to assume that he/she must be knowing what android studio is

File: CONTRIBUTING.md (hunk 2 above)
Comment (ribhavsharma) [reply CONTRIBUTING.md]: If someone wishes to contribute to the Android client, I think it's safe to assume that he/she must be knowing what android studio is

File: CONTRIBUTING.md (hunk 1 above)
Comment (ribhavsharma) [reply CONTRIBUTING.md]: Thanks for pointing out the typo. I'll be fixing this shortly.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+## Finding an Issue
+Now that you've created a local repository, the next step is to find an issue to work on. You can either choose an existing issue or create an Issue ticket of your own.  
+All the Issues for the Android Client can be found [here](https://issues.openmrs.org/projects/AC/issues).   
+Please note that to gain access to the Issue Tracker, you would first have to submit a ticket to the helpdesk. For more details, head over to [this link](https://wiki.openmrs.org/display/docs/How+to+get+access+to+issues.openmrs.org)
+If the issue is tagged ```Ready for Work```, you may claim it and start working on it.
+
+## Working on your local repository
+* First, open Git Bash and ```cd``` into your repository folder.
+* It is common practice to create a branch for every pull request. 
+* To create one, type ```git checkout -b [branch-name]```
```
Comment (f4ww4z) [inline CONTRIBUTING.md]: `branch-name` should be the issue id they're working on. E.g. `AC-520`, `AC-500`

**Pull request merged**